e2e/02-login-flow.e2e.spec.ts

```diff
@@ -101,15 +98,22 @@ describe("Login Flow", () => {
     const backButtonOnAccountScreen = await $(goBack())
     await backButtonOnAccountScreen.waitForDisplayed({ timeout })
     await backButtonOnAccountScreen.click()
-    await browser.pause(1000)
   })
 
   it("navigates back to move home screen", async () => {
-    await scrollUp()
-    const phoneSetting = await $(selector(LL.common.phoneNumber(), "StaticText"))
-    await phoneSetting.waitForDisplayed({ timeout })
+    const maxScrolls = 5
+    const testID = LL.common.phoneNumber()
+    let element = await $(`~${testID}`)
+    for (let i = 0; i < maxScrolls; i++) {
+      if (await element.isDisplayed()) {
```

Comment:
if is typically an antipattern for tests: 
https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-if.md